examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -154,4 +154,27 @@ export interface ITaskList extends IEventProvider<ITaskListEvents> {
 	// readonly handleExternalMessage: (message) => void;
 }
 
+/**
+ * Events emitted by {@link IBaseDocumentEvents}.
+ */
+export interface IBaseDocumentEvents extends IEvent {
+	/**
+	 * Emitted when task list collection has changed.
+	 */
+	(event: "taskListCollectionChanged", listener: () => void);
+}
+export interface IBaseDocumentInitialState {
+	externalTaskListId: string;
+}
```

Comment:
I think this refers more to initial state of a task list than of the base document. Also, is this where other things like the customer service URL will go? Because my instincts want to just pass the task list id instead of a bag that wraps it, unless this is already prep work.

Reply:
Ah my next PR passes the container URL through here as well! So yes, this is a "bag of things".